### ISSUE TITLE
/home is not symlinked to /var/home in flatpaks run on Fedora Silverblue

### ISSUE THREAD

**Issue by kghorvath** [flatpak/flatpak#3582]
## Linux distribution and version
Fedora SIlverblue 32
## Flatpak version
1.6.3
## Description of the problem
When running a flatpak on Silverblue, `$HOME` is created as `/var/home/$USER`. However, there is no symlink created for `/home/$USER`, breaking some applications.
## Steps to reproduce
1.`$ flatpak run --command=bash bar.foo.app`

2.`$ stat /home`
`stat: cannot stat '/home': No such file or directory`

**Comment by vinzent** [comment]
Faceing same issue since re-installed my Fedora Silverblue 33 from scratch today (to reformat with btrfs). Was fine on Fedora Silverblue 32.

**Comment by hfiguiere** [comment]
it's not a bug.
There is nowhere that say the home directory is in `/home`.

**Comment by chrisawi** [comment]
> There is nowhere that say the home directory is in `/home`.

That's true, but it would still be nice to match the host for consistency.

I have a browser profile that was originally created on Fedora Workstation where a `/home/` path got saved in a configuration file. It continued to work on Silverblue due to the symlink, but then it broke when I migrated the browser to Flatpak.

**Comment by rusty-snake** [comment]
> There is nowhere that say the home directory is in `/home`.

- https://www.freedesktop.org/software/systemd/man/latest/file-hierarchy.html
- https://www.man7.org/linux/man-pages/man7/hier.7.html
-https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s08.html

While nothing says it must be there, it is usually to be found there.
